src/registrar/views/utility/mixins.py

```diff
@@ -347,7 +342,6 @@ def has_permission(self):
 
 
 class ApplicationWizardPermission(PermissionsLoginMixin):
-
```

Comment:
(Q) Are these intentional for consistency reasons?

Reply:
This was done by the linter.  Not entirely sure why it is suddenly making changes to all these files...